.github/workflows/ci.yml

```diff
@@ -25,10 +25,29 @@ jobs:
           toolchain: ${{matrix.toolchain}}
           override: true
 
-      - name: Set up protoc
-        uses: arduino/setup-protoc@v1.1.2
+      - name: Test the core crate (akd_core)
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --package akd_core
+
+      - name: Test the core crate (akd_core) with no standard library
+        uses: actions-rs/cargo@v1
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          command: test
+          args: --package akd_core --features nostd,blake3,vrf --no-default-features
+
+      - name: Test the core crate (akd_core) with protobuf and serde enabled
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --package akd_core --features protobuf,serde_serialization
+
+      - name: Test the core crate (akd_core) with no standard library
```

Comment:
Nit: Specify that sha hashing is enabled, as currently the stage name is identical to the one above